src/components/App/index.js

```diff
@@ -1 +1 @@
-export { default } from './App';
```

Comment:
I'm unsure about using absolute imports when the imported is within the same folder, I'm curious, why are you choosing to do so?

Reply:
Consistency mainly but also it makes refactoring easy. Moving things around and copy pasting, becomes less error-prone

Reply:
For me is an easy way to see dependants and if you move the parent (or child) around you should either move them or rename the import. I'm still on the fence, I might bring this up on today's council, people seems to be siding with your take though. Good PR BTW, many relevant improvements.

Reply:
I understand your point and I think is mainly a matter of taste so both approaches are valid. Let me know if you want me to change something here 🤙🏻